### ISSUE TITLE
Certificate Error Loading Stream

### ISSUE THREAD

**Issue by syoung-r7** [Cigaras/IPTV.bundle#75]
### I am getting a certificate error when trying to load a stream. The specific error is below:
```
2017-02-09 08:56:08,880 (7f7273fff700) :  CRITICAL (runtime:889) - Exception (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-b46e0ea/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 843, in handle_request
    result = f(**d)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-b46e0ea/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/handlers/base.py", line 111, in call
    result = self.func(*args, **kwargs)
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/IPTV.bundle/Contents/Code/__init__.py", line 71, in MainMenu
    return ListItems(unicode(L('All')))
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/IPTV.bundle/Contents/Code/__init__.py", line 106, in ListItems
    summary = summary
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/IPTV.bundle/Contents/Code/__init__.py", line 132, in CreateVideoClipObject
    key = GetVideoURL(url = url),
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/IPTV.bundle/Contents/Code/__init__.py", line 297, in GetVideoURL
    return HTTPLiveStreamURL(url = GetRedirectIfPresent(url))
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/IPTV.bundle/Contents/Code/__init__.py", line 324, in GetRedirectIfPresent
    redirect = opener.open(request)
  File "/data/jenkins/pms-deps-universal-v2/BUILD_TAG/linux-ubuntu-x86_64/output/pms-depends-linux-ubuntu-x86_64-release-11a0c7d/lib/python2.7/urllib2.py", line 429, in open
  File "/data/jenkins/pms-deps-universal-v2/BUILD_TAG/linux-ubuntu-x86_64/output/pms-depends-linux-ubuntu-x86_64-release-11a0c7d/lib/python2.7/urllib2.py", line 447, in _open
  File "/data/jenkins/pms-deps-universal-v2/BUILD_TAG/linux-ubuntu-x86_64/output/pms-depends-linux-ubuntu-x86_64-release-11a0c7d/lib/python2.7/urllib2.py", line 407, in _call_chain
  File "/data/jenkins/pms-deps-universal-v2/BUILD_TAG/linux-ubuntu-x86_64/output/pms-depends-linux-ubuntu-x86_64-release-11a0c7d/lib/python2.7/urllib2.py", line 1241, in https_open
  File "/data/jenkins/pms-deps-universal-v2/BUILD_TAG/linux-ubuntu-x86_64/output/pms-depends-linux-ubuntu-x86_64-release-11a0c7d/lib/python2.7/urllib2.py", line 1198, in do_open
URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)>

2017-02-09 08:56:08,883 (7f7273fff700) :  DEBUG (runtime:88) - Sending packed state data (105 bytes)
2017-02-09 08:56:08,883 (7f7273fff700) :  DEBUG (runtime:924) - Response: [500] 2517 bytes
```

**Comment by Cigaras** [comment]
This should be fixed in version [2.0.0](https://github.com/Cigaras/IPTV.bundle/tree/beta) (no release yet, You can download [beta.zip](https://github.com/Cigaras/IPTV.bundle/archive/beta.zip) and rename it to IPTV.bundle).

If issue persists please provide an example stream url for me to test.

**Comment by syoung-r7** [comment]
thanks this fixed it for me!
